js/base-tool.js

```diff
@@ -549,6 +549,149 @@ function baseTool () {
 				});
 			},
 		},
+		{
+			name: "Mass-Modify Pages",
+			desc: "Quickly modify multiple pages.",
+			html: `
+				<div id="d20plus-mass-page-modify" title="Better20 - Mass-Modify Pages">
+					<div name="modify-pages-list">
+						<div class="list" style="transform: translateZ(0); max-height: 490px; overflow-y: scroll; overflow-x: hidden; margin-bottom: 10px;"><i>Loading...</i></div>
+					</div>
+					<hr>
+                    <p><label class="ib"><input type="checkbox" class="select-all"> Select All</label></p>
+					<table name="modifications-table" style="border-collapse: separate; border-spacing: 10px;">
+					<thead>
+						<tr>
+							<th>Overwrite</th>
+							<th>Label</th>
+							<th>Input</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr>
+							<td><input type="checkbox" name="apply_grid_type"></td>
+							<td><label for="grid_type">Grid Type:</label></td>
```

Comment:
imo these labels should better match those in the page edit UI

e.g. `General - Grid - Type` for this; `Lighting - Update when Token Drop` instead of `Light Update Drop`; etc.

Reply:
Thanks for the detailed suggestion, done